install_config/install/deploy_router.adoc

```diff
@@ -1211,73 +1211,82 @@ endif::[]
 
 Save this content to a file for use as the basis of your customized template.
 
-[[using-configmap-replace-template]]
 
+[[using-configmap-replace-template]]
 === Using a ConfigMap to Replace the Router Configuration Template
 
-A xref:../../dev_guide/configmaps.adoc#dev-guide-configmaps[configMap] permits you to customize
-the router instance without rebuilding the router image. The
-*_haproxy-config.template_*, *_reload-haproxy_* and other scripts can be modified
-as well as creating and modifying router environment variables.
-
-The following example shows how to use a custom *_haproxy-config.template_* in
-your router.
+A xref:../../dev_guide/configmaps.adoc#dev-guide-configmaps[ConfigMap] permits
+you to customize the router instance without rebuilding the router image. The
+*_haproxy-config.template_*, *_reload-haproxy_* and other scripts can be
+modified as well as creating and modifying router environment variables.
```

Comment:
@ahardin-rh I know this para was as-is when you got here, but I thin 'permit' has different connotations than what's intended here. Maybe "You can use ConfigMaps to customize the router instance without rebuilding the router image."

Also, oxford comma after "reload-haproxy".